### PR TITLE
[NOMERGE] Request class/views/navigation updates

### PR DESCRIPTION
**Pull request by lkhomenk**
Rhcfqe 2701

Purpose or Intent
=================

__Fixing__ some test had deprecated asserts in requests which was check for last_message == "Request completed" which was removed starting from 5.6

__Extending__ Requests Navmazing a bit + adding Views for widgetastic

and

__Updating tests__ that were using request to use Request class object and it's methods

{{pytest: -v -k "test_provision" --long-running --use-provider vsphere6-nested --use-provider vsphere65-nested}}

### CODE REVIEW

File: cfme/services/requests.py [hunk 1]
```diff
@@ -218,69 +66,270 @@ def find_request(cells, partial_check=False):
...
+    def load_details(self):
+        return navigate_to(self, 'Details')
+
+    def reload(self):
+        toolbar.refresh()
+
+    def update_row(self):
+        self.row = find_request(self.cells, self.partial_check)
+
+    def approve_request(self, reason, cancel=False):
```
Comment (lkhomenk) [inline cfme/services/requests.py]: left as it was before - maybe it would be better to leave only class method and remove view method.

**Comment by dajoRH** [comment]
Lint report for commit 39c8dce6611db1e75a5f3e8d5b9cef0b91265049:

`cfme/services/requests.py`:
- :red_circle: Line 222:11: **N801** *class names should use CapWords convention*
- :red_circle: Line 227:11: **N801** *class names should use CapWords convention*
- :red_circle: Line 231:11: **N801** *class names should use CapWords convention*
- :red_circle: Line 235:11: **N801** *class names should use CapWords convention*
- :red_circle: Line 239:11: **N801** *class names should use CapWords convention*
- :red_circle: Line 243:11: **N801** *class names should use CapWords convention*
- :red_circle: Line 247:11: **N801** *class names should use CapWords convention*
- :red_circle: Line 251:11: **N801** *class names should use CapWords convention*
- :red_circle: Line 255:11: **N801** *class names should use CapWords convention*
- :red_circle: Line 259:11: **N801** *class names should use CapWords convention*

`cfme/tests/services/test_operations.py`:
- :heavy_exclamation_mark: Line 125:38: **W292** *no newline at end of file*

Hmmm
:frowning: You might like to check the lint of your code :point_up: 


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/services/requests.py [hunk 2]
```diff
@@ -218,69 +66,270 @@ def find_request(cells, partial_check=False):
...
+    reload = Button(title='Reload the current display')
+    approve = Button(title='Approve this Request')
+    deny = Button(title='Deny this Request')
+
+
+class RequestDetailsView(RequestView):
+    # TODO Add Request details elements
+    @View.nested
+    class details(View):
+        # Form ? or just test values
```
Comment (lkhomenk) [inline cfme/services/requests.py]: currently this is not used by anyone. but it might be

**Comment by dajoRH** [comment]
Lint report for commit d35bac53453ce977bba133871b7e83b7536006cc:

`cfme/services/requests.py`:
- :red_circle: Line 222:11: **N801** *class names should use CapWords convention*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 0ab54a528e0fe4bb053499f7ad1ea4e8ad3baf8c:

`cfme/services/requests.py`:
- :bangbang: Line 240:5: **E303** *too many blank lines (2)*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit dd1aa32ac2dfd48849bdafb4f169a7e968476929:

`cfme/services/requests.py`:
- :red_circle: Line 6:1: **F401** *'widgetastic_manageiq.CheckboxSelect' imported but unused*
- :bangbang: Line 240:5: **E303** *too many blank lines (2)*
- :bangbang: Line 285:16: **E225** *missing whitespace around operator*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 8417d71c1781722ed524b06ee074170b113f3578:

`cfme/services/requests.py`:
- :red_circle: Line 6:1: **F401** *'widgetastic_manageiq.CheckboxSelect' imported but unused*
- :bangbang: Line 227:5: **E303** *too many blank lines (2)*
- :bangbang: Line 272:16: **E225** *missing whitespace around operator*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by lkhomenk** [comment]
splitting PR into few smaller

**Pull request closed without merging**